src/Workspaces/CoreTestUtilities/ObjectExtensions.cs

```diff
@@ -7,9 +7,9 @@
 using System;
 using System.Reflection;
 
-namespace Microsoft.VisualStudio.IntegrationTest.Utilities.InProcess.ReflectionExtensions
+namespace Microsoft.CodeAnalysis.UnitTests
 {
-    internal static class ObjectExtensions
+    public static class ObjectExtensions
```

Comment:
☹️ Please no crazy extension methods on `object`. It was fine to ignore them previously because they were buried in an edge case namespace and only used by one project.

Reply:
And src/Workspaces/Core/Portable/Serialization/SerializationExtensions.cs too...since this is test only I don't terribly have a problem with this. The alternative is write the crazy multiple times.

Reply:
An acceptable resolution here is changing the extension methods to normal static methods. The calling sequence is slightly different, but it does a good job of localizing the pain.